src/components/Scope/index.js

```diff
@@ -61,7 +67,7 @@ export const Scope = ({ children, filterPlaceholder }) => {
 				reset={reset}
 				getScope={getScope}
 				nodeState={nodeState}
-				defaultNodeState={defaultNodeState}
+				defaultNodeState={opening ? defaultNodeState : {}}
```

Comment:
This is better done as a modification or filter on the `nodeState` field. The scope selector maintains state between unmount and mount through this - as a feature of the TreeView component. If you were to set an effect on it where you reset the nodeState field when unmounting (cf. [`useEffect` cleanup](https://reactjs.org/docs/hooks-effect.html#effects-with-cleanup)), for example, that would drop the persistent state and go back to the default. Other options exist for solving this in different ways.